app/config/default.json

```diff
@@ -1,4 +1,5 @@
 {
+  "version": "1.7.0",
```

Comment:
This version (and the bower.json version) is updated by grunt when you run npm version <major/minor/patch>:
[npm script](https://github.com/jembi/openhim-console/blob/ohm-254-add-display-version/package.json#L53)
[grunt task](https://github.com/jembi/openhim-console/blob/ohm-254-add-display-version/Gruntfile.js#L339)